packages/utils/upgrade/src/modules/version/range.ts

```diff
@@ -19,6 +22,14 @@ export const rangeFromReleaseType = (current: Version.SemVer, identifier: Versio
       const nextMajor = semver.inc(current, 'major') as Version.LiteralSemVer;
       return fromCurrentTo(nextMajor);
     }
+    case Version.ReleaseType.Patch: {
+      const preMinor = semver.inc(current, 'preminor') as Version.LiteralSemVer;
```

Comment:
Could you add a small comment with an example of what inc(version, 'preminor') does like what is done for major? (same for minor)
It could help with reading through the code later on